src/menus/optionsMenu.cpp

```diff
@@ -173,8 +173,10 @@ OptionsMenu::OptionsMenu()
     (new GuiLabel(interface_page, "LANGUAGE_OPTIONS_LABEL", tr("Language (applies on back)"), 30))->addBackground()->setSize(GuiElement::GuiSizeMax, 50);
     
     //Don't wan't scripts/locale. Also, compatible with android assets.
-    DirectoryResourceProvider* lang_prov = new DirectoryResourceProvider("resources/locale");
+    DirectoryResourceProvider* lang_prov {new DirectoryResourceProvider("resources/locale")};
```

Comment:
is there a reason you're not using a variable on the stack directly?

Reply:
Yes, see p.h line 6 to 14.